filters/all.js

```diff
@@ -531,6 +539,29 @@ function getAdditionalSubs(asyncapi, params) {
   return ret;
 }
 
+function getBrokerSettings(asyncapi,params){
+ let ret;
```

Comment:
what `ret` stands for?

Reply:
It was for returning a value.. ill refactor this to a proper name.

Reply:
I'm trying to run this. I grabbed a copy of your fork, Amrut, and switched to this branch. When I ran it I got an error saying that it didn't accept the useServers parameter, even though I see it clearly in your package.json. I assume this works for you?